pkg/apis/app/v2/kieapp_types.go

```diff
@@ -942,6 +944,26 @@ type ObjectReference struct {
 	FieldPath string `json:"fieldPath,omitempty" protobuf:"bytes,7,opt,name=fieldPath"`
 }
 
+// KafkaExt kafka configuration to be used by the KieApp
+type KafkaExt struct {
+	// Disable the Kafka Extension
+	ExtDisabled string `json:"extDisabled,omitempty"`
+	// A comma separated list of pairs <message/signal name>=<topicName>
+	Topics string `json:"topics,omitempty"`
```

Comment:
would be better have a list instead a string here?
then during env provisioning you read this env and properly configure it.

Reply:
No, because they wait for a string with all topics configurations pairs, is up to the java code splitting with comma